src/test/java/cubecart/utils/SQLQueries.java

```diff
@@ -0,0 +1,62 @@
+package cubecart.utils;
+
+import javax.sql.rowset.CachedRowSet;
+import javax.sql.rowset.RowSetProvider;
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+/**
+ * @author : alfir
+ * @created : 3/3/2024,2:43 PM
+ **/
+public class SQLQueries {
+    public boolean getProductInfo(String productName, Connection connection) {
+        boolean isProductExist = false;
+        Statement statement = null;
+        ResultSet resultSet = null;
+        CachedRowSet cachedRowSet = null;
+        try {
+            cachedRowSet = RowSetProvider.newFactory().createCachedRowSet();
+            statement = connection.createStatement();
+        } catch (SQLException s) {
+            s.printStackTrace();
+        }
+        String sqlScript = String.format
+                ("select product_id, name, price from cc_CubeCart_inventory where name='%s'", productName);
+        try {
+            resultSet = statement.executeQuery(sqlScript);
+        } catch (SQLException e) {
+            throw new RuntimeException(e);
```

Comment:
this class can be deleted. since its already added into DButil

Reply:
Deleted.